lib/Versions

```diff
@@ -57,6 +57,7 @@ LIBNXZ_0.61.0 {
 };
 
 LIBNXZ_0.63.0 {
+    nx_crc32_z;
```

Comment:
what is the goal of exporting `nx_crc32_z` if it's identical to `nx_crc32`?

Reply:
Now that `nx_crc32` and `nx_crc32_z` are the same, there's really no point. I'll remove it.